packages/react/src/components/NumberInput/NumberInput.js

```diff
@@ -289,8 +289,8 @@ class NumberInput extends Component {
             onClick && onClick(evt, { value, direction });
             onChange && onChange(evt, { value, direction });
           } else {
-            onClick && onClick(evt, direction);
-            onChange && onChange(evt, direction);
+            onClick && onClick(evt, { value, direction });
+            onChange && onChange(evt, { value, direction });
```

Comment:
I believe this would be a breaking change since the function signature is changing so you probably want to keep the existing signature and add to it instead of changing its shape

Reply:
ah, makes sense. Will update! 👍🏽